tictactoe.rb

```diff
@@ -0,0 +1,128 @@
+class Game
+    attr_accessor :board, :board_o, :board_x, :b
+
+    def initialize
+        #to intialize -start method and winning positions
+        start
+       
+        @win_positions =
+      [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+    end
+
+    def start
+        #@turns_played = 0
+         @player = "x"
+        @board = ["-", "-", "-", "-", "-", "-", "-", "-", "-"]
+        @oard_o=0
+        @board_x=0
+        @b=0
+    end
+    def show_board(board = @board)
```

Comment:
Two things here:
* `@board` is an instance variable of the `Game` class, so there's no need to declare `board = @board`. The idea here is that you're passing in an Array of 3 arrays with a `-`. This could just be `show_board(@board)`.

Reply:
2. This method puts three Arrays that represent a board by concatenating each cell in a row every time it is called inside your `play` method. Is there a better way to accomplish this? 

Part of the answer is perhaps making your `@board` instance variable generate the board in an Array of 3 Arrays, e.g. `@board = Array.new(3) { Array.new(3) { '-' } }`.